docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -62,7 +62,7 @@ The `browser-list-comma-separated` argument specifies the list of browsers (sepa
 ### Local Browsers
 
 You can specify [locally installed browsers](common-concepts/browsers/browser-support.md#locally-installed-browsers) using browser aliases or paths (with the `path:` prefix).
-The list of all the available browsers can be obtained by calling the [--list-browsers](#-b---list-browsers) command.
+Use the [--list-browsers](#-b---list-browsers) command to find browsers that can be automatically detected and referred to by their aliases.
```

Comment:
Maybe, I was thinking about it

Reply:
If only... This is the first place we mention auto-detected browsers. In my version, we implicitly describe what these browsers are (these are browsers that can be detected and referred to by their alias). In your version, we just mention `'auto-detected browsers'`...

Reply:
As we are in the Local Browsers section, we can write: Use the ... command to output aliases for locally installed browsers.

Reply:
We can only find automatically detected browsers, not all locally installed. But you're right, other sentences in this section already say what you can do with the aliases, so that it's redundant to say it once again.